recipes/icu/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "56.1" %}
+
+package:
+  name: icu
+  version: {{ version }}
+
+source:
+  fn:  icu4c-{{ version.replace(".", "_") }}-src.tgz
+  url: http://download.icu-project.org/files/icu4c/{{ version }}/icu4c-{{ version.replace(".", "_") }}-src.tgz
```

Comment:
I think I tried this with Boost too. Other people complained it was a little ugly and hard to read. I think they convinced me. Could we add a another variable and assign the result from `version.replace(".", "_")` to it? It could either go up top or maybe the top of this section. Thoughts?